### ISSUE TITLE
[Android] Add advanced setting to set max frequency

### ISSUE THREAD

**Issue by VanceVagell** [VanceVagell/kv4p-ht#18]
In the Settings page, add a new "Advanced" sub-section after "Accessibility". In that new section, add a "Max frequency" drop-down with the options "148MHz" (which should be selected by default) and "146MHz".  Some countries limit their 2M frequencies to 144-146MHz, and this will let users restrict their device so they don't accidentally tx outside their permissions.

If the frequency is limited to 146MHz and the currently-tuned frequency is above that, tune to "146.000" immediately.

Note that this setting will also need to be added and saved in the app's internal DB with other settings.

**Comment by DrFrankReade** [comment]
Consider adding non-standard repeater offsets to advanced settings (These are edge cases I realize, I only have 2 out of 40 or so repeaters in my area that don't use a standard split)
